apps/system/js/notifications.js

```diff
@@ -122,6 +126,86 @@ var NotificationScreen = {
     }, function(err) {
       console.error('VersionHelper failed to lookup version settings.');
     });
+
+
+  },
+
+  getStore: function ns_getStore() {
+    return new Promise(resolve => {
```

Comment:
It seems to me, that Promise constructor shall accept 2 parameters ``resolve`` & ``reject``, also it would be really nice, if you reject your new Promise in case of rejection of ``getDataStores()`` call ;-)
BTW, absence of second parameter accompanied with absence of ``'('`` & ``')'`` around arrow function's parameter could (potentially) lead to problems with lint.